umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10/data_contact.clj

```diff
@@ -16,7 +16,7 @@
   "Investigator" "INVESTIGATOR"
   "Metadata Author" "METADATA AUTHOR"
   "User Services" "User Services"
-  "Science Software Development" "Science Software Development Manager"})
+  "Science Software Development" "Science Software Development"})
```

Comment:
In the ticket, it asked for "Technical Contact for Science to Science Contact"  Does it no longer apply? Have you clarify with @eereiter?

Reply:
Actually I see it "Science Contact" "Technical Contact for Science" on line 14, Should change the value to "Science Contact"?

Reply:
done